Readme.md

```diff
@@ -3,29 +3,59 @@
 [![](https://img.shields.io/badge/Open_in_DevExpress_Support_Center-FF7200?style=flat-square&logo=DevExpress&logoColor=white)](https://supportcenter.devexpress.com/ticket/details/E5197)
 [![](https://img.shields.io/badge/📖_How_to_use_DevExpress_Examples-e9f6fc?style=flat-square)](https://docs.devexpress.com/GeneralInformation/403183)
 <!-- default badges end -->
-<!-- default file list -->
-*Files to look at*:
+# Upload Control for ASP.NET Web Forms - How to upload an image and display it on a web page
 
-* [Default.aspx](./CS/WebSite/Default.aspx) (VB: [Default.aspx](./VB/WebSite/Default.aspx))
-* [Default.aspx.cs](./CS/WebSite/Default.aspx.cs) (VB: [Default.aspx.vb](./VB/WebSite/Default.aspx.vb))
-<!-- default file list end -->
-# How to use ASPxUploadControl to upload an image and display it on a web page
+This example demonstrates how to use the [Upload Control](https://docs.devexpress.com/AspNet/8298/components/file-management/file-upload) to upload an image to the server and display the uploaded image on a web page.
+
+![Upload and Display an Image](upload-image.png)
+
+## Overview
+
+Follow the steps below to display an image on a web page once a user uploads the image to the server:
+
+1. Create the *Images* folder in the root folder of your project. In the newly created folder, place an image that you want to initially display.
```

Comment:
Word order -- "initially" goes to the end of the base sentence as supplement (an aspect of circumstances that answers the question "how"):

```suggestion
1. Create the *Images* folder in the root folder of your project. In the newly created folder, place an image you want to display initially.
```